tests/test_bpi_scrapping.py

```diff
@@ -39,6 +39,16 @@ def test_get_article_title_link(bpi_data_soup, bpi_scrapper):
         "https://www.bpifrance.fr/nos-appels-a-projets-concours/appel-a-projets-france-chili-greentech",
     )
 
+def test_get_article_link(bpi_data_soup, bpi_scrapper):
+    articles = bpi_scrapper.get_articles(bpi_data_soup)
+    _, link = bpi_scrapper.get_article_title_and_link(articles[0])
+    assert link == "https://www.bpifrance.fr/nos-appels-a-projets-concours/appel-a-projets-france-chili-greentech"
+
+def test_get_article_full_content(bpi_data_soup, bpi_scrapper):
```

Comment:
how do you test nominal cases, edge cases?
ie what should happen if the article link leads to a 404 (if the article has been removed by BPI)?

Reply:
@ldt does testing the error is thrown and well executed ok ?